bundle/manifests/kmm-operator.clusterserviceversion.yaml

```diff
@@ -239,8 +247,8 @@ spec:
                 - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
                 command:
-                - /manager
-                image: quay.io/edge-infrastructure/kernel-module-management:latest
+                - /usr/local/bin/manager
+                image: quay.io/quba/kmmo:latest
```

Comment:
Why do you point to your quay account ? Maybe is from testing ?

Reply:
Should be fixed - apologies